docs/source/core/partition.rst

```diff
@@ -872,13 +872,14 @@ settings supported by the API.
   filename = "example-docs/DA-1p.pdf"
 
   elements = partition_via_api(
-    filename=filename, api_key=api_key, strategy="auto", pdf_infer_table_structure="true"
+    filename=filename,
+    api_key=api_key,
+    strategy="auto",
+    pdf_infer_table_structure="true"
   )
 
-If you are self-hosting or running the API locally, you can use the ``api_url`` kwarg
-to point the ``partition_via_api`` function at your self-hosted or local API.
-See `here <https://github.com/Unstructured-IO/unstructured-api#dizzy-instructions-for-using-the-docker-image>`_ for
-documentation on how to run the API as a container locally.
+If you use Unstructured SaaS API or running the API locally, you can use the ``api_url`` kwarg
+to point the ``partition_via_api`` function at `Unstructured SaaS API <https://unstructured-io.github.io/unstructured/apis/saas_api.html>`__ or `self-hosted API <https://github.com/Unstructured-IO/unstructured-api>`__.
```

Comment:
```suggestion
to point the ``partition_via_api`` function at your `Unstructured SaaS API <https://unstructured-io.github.io/unstructured/apis/saas_api.html>`__  URL, or the `self-hosted API <https://github.com/Unstructured-IO/unstructured-api>`__.
```

Reply:
wait im not sure if the formatting worked there actually. can we just change have this say "point ... at your Unstructured SaaS API URL" instead of just "Unstructured SaaS API"?

Reply:
I've updated the wording by removing the links and refer to SaaS or Self-Hosted API URL.